lib/utils/hasANPlusBNotationPseudoClasses.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const {
+	aNPlusBNotationPseudoClasses,
+	aNPlusBOfSNotationPseudoClasses,
+} = require('../reference/selectors');
+
+const HAS_A_N_PLUS_B_NOTATION_PSEUDO_CLASSES = new RegExp(
+	`\\b:(?:${[
+		...aNPlusBNotationPseudoClasses.values(),
+		...aNPlusBOfSNotationPseudoClasses.values(),
+	].join('|')})\\(`,
+	'i',
+);
```

Comment:
[suggest] How about using a local variable to improve readability for the regex like this?

```suggestion
const classes = [
	...aNPlusBNotationPseudoClasses.values(),
	...aNPlusBOfSNotationPseudoClasses.values(),
].join('|');
const HAS_A_N_PLUS_B_NOTATION_PSEUDO_CLASSES = new RegExp(`\\b:(?:${classes})\\(`, 'i');
```

Reply:
yes, much better, thank you 🙇